pages/accountLists/[accountListId]/tools.page.tsx

```diff
@@ -0,0 +1,31 @@
+import React, { ReactElement, useEffect } from 'react';
+import { useTranslation } from 'react-i18next';
+import { useRouter } from 'next/router';
+import Head from 'next/head';
+import { useApp } from '../../../src/components/App';
+import ToolHome from 'src/components/Tool/Home/Home';
+
+const ToolsPage = (): ReactElement => {
+  const { dispatch } = useApp();
+  const { t } = useTranslation();
+  const router = useRouter();
+
+  useEffect(() => {
+    dispatch({ type: 'updateBreadcrumb', breadcrumb: t('Tools') });
+    dispatch({
+      type: 'updateAccountListId',
+      accountListId: router.query.accountListId?.toString() ?? '',
+    });
+  }, []);
```

Comment:
Just toss this. We're moving away from a global app state and I don't  think the breadcrumb is used anywhere. If you need the `accountListId`, use the `useAccountListId` hook. If we need some global state in the future, we can make dedicated hooks for the specific 
 feature needed.

Reply:
Done